models/pose_estimation_mediapipe/demo.cpp

```diff
@@ -0,0 +1,2913 @@
+#include <vector>
+#include <string>
+#include <utility>
+#include <cmath>
+
+#include <opencv2/opencv.hpp>
+const long double M_PI = 3.141592653589793238L;
```

Comment:
Seems like `M_PI` is a macro of some platforms, can we just change the name to like `_M_PI`?

Reply:
Ok 
#include <opencv2/core/softfloat.hpp>

const long double _M_PI = cv::softdouble::pi();

Reply:
I think just `const long double _M_PI = 3.141592653589793238L;` is enough.

Reply:
OK done